boards/others/supermini_nrf52840/board.yml

```diff
@@ -0,0 +1,7 @@
+board:
+  name: supermini_nrf52840
```

Comment:
My understanding is that Supermini is a form factor the same way Pro Micro is a form factor. Supermini is not a manufacturer. There is a separate [Supermini nRF52840](https://www.aliexpress.com/item/1005006974687017.html) the same way there is Supermini ESP32-C3, etc.

[This](https://www.nologo.tech/product/otherboard/NRF52840.html) like the most authoritative page I've been able to find so I would consider "Nologo" the vendor.

Reply:
Please look at the discussion after your messages in https://github.com/zephyrproject-rtos/zephyr/pull/80100 and at the namings in https://github.com/joric/nrfmicro/wiki/Alternatives.
'supermini' is not a form factor, and it's not intended as a manufacturer, 'pro micro' is the form factor. Meanwhile the nologo board uses the nice!nano bootloader and says it's a 'pro micro' format board on the page you linked. You are free to explain the naming on this: https://www.aliexpress.com/item/1005007383270623.html and the fact typing 'supermini nrf52840' in aliexpress or google returns 'pro micro' form factor boards. The 'ICBuy Supermini' looks like it came later to any of the pro micro 'supermini' and is not pointed at by google when typing 'supermini nrf52840'. Consider also this page: https://circuitpython.org/board/supermini_nrf52840/

This is intended to cover for multiple 'brands', mostly of clones (in my personal case, of tenstar making, which is also called 'supermini' in the wiki, nice!nano compatible and 'pro micro' in the aliexpress entry, after a black colored one that looked more like the nologo one failed to deliver) so it will not go in nologo or in a nicekeyboard brand folder, as discussed in the previous PR (because then the 'original' is the nice!nano).

Reply:
Actually it's even more confusing than that, here is 'ICBuy supermini': https://www.aliexpress.com/item/1005006019812115.html
and also, 'ICBuy supermini': https://www.aliexpress.com/item/1005006974687017.html

So I believe ICBuy's naming can be safely ignored and there is no common nrf52840 'supermini' (as in ESP32-C3 supermini) format.

Reply:
I think there is a case to be made for having the vendor "others" rather than "Nologo" so it covers both nice!nano along with clones.

Supermini nRF52840 was what the board was called when it was initially released. However that was eventually renamed Pro Micro nRF52840. The [product page](https://www.nologo.tech/product/otherboard/NRF52840.html) directly says this:

> 2023.9.28之后名称由SuperMiniNRF52840改为ProMicro52840

Supermini now describes a different line of products, including boards such as [Supermini RP2040](https://www.nologo.tech/product/raspberrypie/RP2040SuperMini.html).

Reply:
Then are you suggesting renaming it back to promicro but keeping it in others?

Reply:
If multiple vendors make this, put it in others board and put others for the vendor name, as seen in https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/others/black_f407ve/board.yml

Reply:
Renamed back to promicro and added vendor: others